pkg/suse/salt-common.logrotate

```diff
@@ -33,3 +33,19 @@
 	compress
 	notifempty
 }
+
+/var/log/salt/syndic {
```

Comment:
Don't you need `su salt salt` here as above?

Reply:
Don't understand the comment 'su salt salt'.
/var/log/salt/syndic is the syndic log in /var/log/salt and already in regular salt, as in, pkg/salt-common.logrotate, and noticed it missing from suse version ().  Adding /var/log/salt/proxy for both regular salt and suse.  The salt-common.logrotate file is used at install time from build packages and is getting installed anyway.

Reply:
I mean all the records in this logrotate config start either from `su salt salt` or from `su root root`. I don't know why, just found it and pointed your attention to this. Sorry if I'm wrong about that.

Reply:
@DmitryKuzmenko Opened up the file and viola, see what you mean, will add the su salt salt, thanks for catching that